LinqToDBPostGisNetTopologySuite/LinearReferencing.cs

```diff
@@ -0,0 +1,175 @@
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Return a linestring being a substring of the input one starting and ending at the given fractions of total 2d length
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_LineSubstring.html
+        /// </remarks>
+        /// <param name="geometry">Input geometry</param>
+        /// <param name="startFraction">Startfraction(Between 0 and 1)</param>
+        /// <param name="endFraction">Endfraction(Between 0 and 1)</param>
+        /// <returns>Linestring being a substring of the input one starting and ending at the given fractions of total 2d length</returns>
+        [Sql.Function("ST_LineSubstring", ServerSideOnly = true)]
+        public static NTSG STLineSubstring(this NTSG geometry, double startFraction, double endFraction)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Return a derived geometry collection value with elements that match the specified measure(Polygonal elements are not supported).Resultant will be offset to the left(When offset is positive) or right(When offset is negative) of the input line by the specified number of units
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_LocateAlong.html
+        /// </remarks>
+        /// <param name="geometry">Input geometry</param>
+        /// <param name="measure">Measure</param>
+        /// <param name="offset">Offset</param>
+        /// <returns>Derived geometry collection value with elements that match the specified measure</returns>
+        [Sql.Function("ST_LocateAlong", ServerSideOnly = true)]
+        public static NTSG STLocateAlong(this NTSG geometry, double measure, double offset)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Return a derived geometry collection with elements that match the specified range of measures inclusively.The resultant will be offset to the left(When offset is positive) or right(When offset is negative) of the input line by the specified number of units
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_LocateBetween.html
+        /// </remarks>
+        /// <param name="geometry">Input geometry</param>
+        /// <param name="measure_start">Measure start</param>
+        /// <param name="measure_end">Measure end</param>
+        /// <param name="offset">Offset</param>
+        /// <returns>Derived geometry collection with elements that match the specified range of measures inclusively</returns>
+        [Sql.Function("ST_LocateBetween", ServerSideOnly = true)]
+        public static NTS.GeometryCollection STLocateBetween(this NTSG geometry, double measure_start, double measure_end, double offset)
```

Comment:
- Returning common type 'NetTopologySuite.Geometries.Geometry' (`NTSG`) would be OK (as all methods are implemented now: PostGIS `geometry` maps to `NetTopologySuite.Geometries.Geometry`)
- `measure_start`, `measure_end` are violating naming convention for C# (should be `measureStart`, `measureEnd`)